packages/stylist-macros/src/output/style_attr.rs

```diff
@@ -1,11 +1,13 @@
-use super::{fragment_coalesce, ContextRecorder, IntoCowVecTokens, OutputFragment, Reify};
+use super::{
+    fragment_coalesce, ContextRecorder, IntoCowVecTokens, OutputCowString, OutputFragment, Reify,
+};
 use itertools::Itertools;
 use proc_macro2::TokenStream;
 use quote::quote;
 use syn::parse::Error as ParseError;
 
 pub struct OutputAttribute {
-    pub key: OutputFragment,
+    pub key: OutputCowString,
```

Comment:
This is not directly related to this PR, but rather discovered when I was working on #31.

Is it useful to make the style key substitutable?

Reply:
The current declaration of `stylist::ast::StyleAttribute` uses a `Cow<'static, str>`, not a `StringFragment`, hence the change to reflect that reality.

```
pub struct StyleAttribute {
    pub key: Cow<'static, str>,
    pub value: Cow<'static, [StringFragment]>,
}
```

Reply:
I do like the idea of making them substitutable, I'm just not sure if we should allow more than one part, e.g. `border-${side}: 5px solid black;` hence I didn't want to include that kind of change here.

Reply:
OK. I guess it's fine to have `OutputCowString` for now.

In the future, maybe the key also needs to be modified to `Cow<'static, [StringFragment]>` to support proper interpolation.